<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenImages"/>
    <addaction name="actionCloseImages"/>
    <addaction name="separator"/>
    <addaction name="actionSaveImageAs"/>
    <addaction name="actionSaveVisibleAreaAs"/>
   </widget>
   <widget class="QMenu" name="menuComparators">
    <property name="title">
     <string>Comparators</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuImageAnalysis">
    <property name="title">
     <string>Image Analysis</string>
    </property>
    <addaction name="actionSwitchBetweenImages"/>
    <addaction name="separator"/>
    <addaction name="actionColorPicker"/>
    <addaction name="actionAdvancedColorPicker"/>
   </widget>
   <widget class="QMenu" name="menuTransftomers">
    <property name="title">
     <string>Transfomers</string>
    </property>
    <addaction name="actionShowOriginalImage"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImageAnalysis"/>
   <addaction name="menuComparators"/>
   <addaction name="menuTransftomers"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpenImages">
   <property name="text">
    <string>&amp;Open Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCloseImages">
   <property name="text">
    <string>&amp;Close Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSwitchBetweenImages">
   <property name="text">
    <string>Switch between images being compared</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSaveImageAs">
   <property name="text">
    <string>Save Image As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveVisibleAreaAs">
   <property name="text">
    <string>Save Visible Area As... </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionColorPicker">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Color Picker</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionShow_R_channel">
   <property name="text">
    <string>Display Red Channel</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionShow_G_channel">
   <property name="text">
    <string>Display Green Channel</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionShow_B_channel">
   <property name="text">
    <string>Display Blue Channel</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionShow_RGB">
   <property name="text">
    <string>DIsplay RGB</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionDisplay_grayscale">
   <property name="text">
    <string>Grayscale </string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionShowOriginalImage">
   <property name="text">
    <string>Show Original Image</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionAdvancedColorPicker">
   <property name="text">
    <string>Advanced Color Picker</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
